calicoctl/calicoctl/commands/crds/crds.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package crds
+package crds 
```

Comment:
@invidian I think you need to re-run the generation, since the extra space is still here

Reply:
Ugh, of course, just looked at the CI logs. I _think_ it should be OK now.